### ISSUE TITLE
Cart Popup Notification can only handle one product, not compatible with bundles

### ISSUE THREAD

**Issue by ct-Marvin** [Shopify/dawn#3114]
## Describe the current behavior
The Cart Popup encounters an error when a bundle that applies the 'merge' cart transform function or when multiple products are added. In contrast, this functionality operates correctly with both the Cart Drawer and the Cart Page.  

Error Message in the console:
`product-form.js?v=183032352727205275661700096255:101 TypeError: Cannot read properties of null (reading 'innerHTML')
    at CartNotification.getSectionInnerHTML (cart-notification.js?v=133508293167896966491700096255:81:86)
    at cart-notification.js?v=133508293167896966491700096255:55:60
    at Array.forEach (<anonymous>)
    at CartNotification.renderContents (cart-notification.js?v=133508293167896966491700096255:54:32)
    at product-form.js?v=183032352727205275661700096255:97:25`

## Describe the expected behavior
The Cart Popup should be able to handle multiple products and bundles beeing added.

- Dawn Version: 12.0.0

**Comment by platanodev** [comment]
Same here!

**Comment by mironov** [comment]
@platanodev Yes, exactly like what you have posted. If several items are added to the cart in one request, the `renderContents` will not be able to process the response.

**Comment by platanodev** [comment]
Correct, we just implemented it as you specified and it works. The only issue is that the cart notification only displays 1 of the 2 products added, did you see that?
